### PR TITLE
Add support for various minecraft files.

### PR DESCRIPTION
**Pull request by Levertion**
Adds support for:
`.mcfunction` files,
`pack.mcmeta` files,
`.png.mcmeta` files.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ----------------------
 ### Added
 - **Support:** GIMP (`.gpl`)
+- **Support:** Minecraft Function (`.mcfunction`)
```
Comment (Alhadis) [inline CHANGELOG.md]: Please merge this into one line:

~~~markdown
- **Support:** GIMP (`.gpl`), Minecraft (`.mcfunction`, `pack.mcmeta`, `.png.mcmeta`)
~~~

The change-log is currently 1421 lines long, and that's with all relevant changes grouped on the same line. Imagine how long the file would become if we split it across lines...

File: config.cson [hunk 2]
```diff
@@ -2701,9 +2701,12 @@ fileIcons:
 
 	Minecraft:
 		icon: "minecraft"
-		scope: "forge-config"
-		match: /^mcmod\.info$/i
-		colour: "dark-green"
+		match: [
+			[/^mcmod\.info$/i,"dark-green",{scope:"forge-config"}]
```
Comment (Alhadis) [inline config.cson]: Please add a space between each comma/colon and the following value.

The curly brackets around `scope:` are also unnecessary and should be removed to be consistent with the rest of the file's formatting.

**Comment by Levertion** [comment]
Updated - is that OK now? I shouldn't need to update config for these stylistic changes.

**Comment by Alhadis** [comment]
Yep, that's fine. Thanks!

**Pull request merged**